inc/switchtec/switchtec.h

```diff
@@ -440,7 +440,11 @@ int switchtec_fw_img_write_hdr(int fd, struct switchtec_fw_footer *ftr,
 int switchtec_fw_is_boot_ro(struct switchtec_dev *dev);
 int switchtec_fw_set_boot_ro(struct switchtec_dev *dev,
 			     enum switchtec_fw_ro ro);
+int switchtec_bind_info(struct switchtec_dev *dev, struct switchtec_bind_status_out *bind_status, int phy_port);
```

Comment:
Max line length is 80 chars. These lines are too long and should be wrapped and indented just like the other functions in this file. (Per Kernel Style)